prql-php/README.md

```diff
@@ -0,0 +1,25 @@
+# prql-php
+
+`prql-php` offers PHP bindings through FFI.
+
+It provides the `Compiler` class which contains the `toJson` and `toSql`
```

Comment:
We'll want to rename `toSql` to `compile` to match the other bindings' APIs, but it's OK to do that later when we update `prql-lib`

Reply:
Yes, I would like to do this too, it is `toSql` just for now until the `prql-lib` gets updated, then I plan to update it to use the new API.